app/scripts/templates/complete_reset_password.mustache

```diff
@@ -57,7 +57,7 @@
         <input type="password" class="password check-password tooltip-below" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" required autofocus />
 
         <input id="show-password" type="checkbox" class="show-password" aria-controls="password" tabindex="-1" data-novalue data-synchronize-show="true" />
-        <label for="show-password" class="show-password-label">
+        <label for="show-password" class="show-password-label hidden">
```

Comment:
On further thought, I'd test a completely different approach. Remove the label entirely from the HTML and only add it whenever we want it to be displayed. This would match a progressive enhancement approach, and further reduces the chance of error whenever creating new password fields.